Pack.tests.template.ps1

```diff
@@ -33,18 +32,21 @@ foreach ($packFile in $packFiles) {
 
             It "Has a details.json" {
                 $packFile.FullName | Should Exist
+                $packFile.Name | Should BeExactly "details.json"
             }
 
             It "Has a Readme.md" {
                 $readme = $packFile.FullName -replace $packFileName, 'README.md'
                 $readme | Should Exist
+                # Get-ChildItem is used as Get-Item returns the same case it is passed.
+                (Get-ChildItem -Path $packFile.DirectoryName -Filter "readme.md").Name  | Should BeExactly "ReadMe.md"
```

Comment:
In VSCode only the test name is surfaced - can you either update it to include this restriction or (preferably) create another test?  Good unit testing practise is to only test a single thing with each test.

(also, feel free to update my incorrectly cased file name above!)

Reply:
To increase the readability of the output I'll change the test name to "Has a ReadMe.md (case-sensitive)", this should help a user that gets the error even through they have a README.md in the folder.